guides/source/active_record_basics.md

```diff
@@ -125,6 +125,12 @@ Here are some more examples of model class names and corresponding table names:
 | `Product`        | `products`     |
 | `Person`         | `people`       |
 
+For models nested inside of a module, only the first parent module will be used as
+a prefix onto the table. It will first be singularized, and then followed by an underscore.
+
+If the module is labeled as `abstract`, then it will
+not be used as a prefix.
+
```

Comment:
Hi, 
Thanks for adding additional notes for better understanding. 
Could you please add an example code snippet for better readability?

Reply:
Done!

Reply:
Thank you!